core/utils/baseRepo.js

```diff
@@ -6,9 +6,10 @@ const addCreatedTimestamps = (payload) => {
   return payload;
 };
 
-const create = async (table, payload) => {
+const create = async (table, payload, timestamp=true) => {
   try {
-    payload = addCreatedTimestamps(payload);
+    if(timestamp)
```

Comment:
@rjvim i added this timestamp condition because in tags table we dont have timestamps and this condition gave sql error. I will add this method in tagRepo for this

Reply:
@rohan099 In this case, you can check code in baseRepo and directly call using "knex". I made baseRepo only with few helper functions, they might not fit all the scenarios. baseRepo assumes that that there are timestamps including deleted_at. Check baseRepo - remove/delete function.

Reply:
ok